shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformChannel.java

```diff
@@ -640,7 +640,10 @@ static SystemUiOverlay fromValue(@NonNull String encodedName) throws NoSuchField
     LEAN_BACK("SystemUiMode.leanBack"),
     IMMERSIVE("SystemUiMode.immersive"),
     IMMERSIVE_STICKY("SystemUiMode.immersiveSticky"),
-    EDGE_TO_EDGE("SystemUiMode.edgeToEdge");
+    EDGE_TO_EDGE("SystemUiMode.edgeToEdge"),
+    MANUAL_TOP("SystemUiMode.manualTop"),
```

Comment:
these values are sent from the framework, https://github.com/flutter/flutter/blob/61a0add2865c51bfee33939c1820709d1115c77d/packages/flutter/lib/src/services/system_chrome.dart#L90

are these new values meant to be used in the embedding only?

Reply:
Yeah, these are just to be used in the embedding. I needed a way to track which `SystemUiMode` was set in the case that `updateSystemUiOverlays` was called.

Reply:
it seems like these options need to be in a new enum. Also, what does manualTop/manualBottom/manualBoth mean? They would need documentation

Reply:
Undid this in lieu of just saving the overlays set, since there is an enum already defined for those.